lib/src/gotrue_client.dart

```diff
@@ -171,6 +171,11 @@ class GoTrueClient {
     return response;
   }
 
+  /// Overrides the JWT on the current client. The JWT will then be sent in all subsequent network requests.
+  Session setAuth(String accessToken) =>
+      currentSession = currentSession?.copyWith(accessToken: accessToken) ??
```

Comment:
Should we set `token_type: 'bearer'` here like how supabase-js is doing [here](https://github.com/supabase/gotrue-js/pull/83/files#diff-3522461172efd6058d6b8da62fc2d30d8b524d2b64894ea2c67218c52f7fdff5R262)?

Reply:
Yes, probably. Just for consistency. But in my search of both the supabase and gotrue libraries (in both dart and js), I could not find that field ever being used. The only time it seems to be checked is just to see if it exists (in js) and then throws if not. So I'm not really sure what it is for. Nonetheless, I will amend the PR. Standby..

Reply:
@point-source Thanks for doing deep research into both client libraries! You are right. It does not seem to be used now, but let's keep it for consistency, and we can deprecate it later if we really need to!

Reply:
Sounds good to me! Thanks